bin/gltf-pipeline.js

```diff
@@ -47,7 +49,8 @@ if (!defined(outputPath)) {
 
 var options = {
     isBinary : isBinary,
-    isEmbedded : !isSeparate
+    isEmbedded : !isSeparate,
+    isImageEmbedded : !isTextureSeparate
```

Comment:
The naming is a bit inconsistent with `texture` and `image`. In the context of gltf, I think we should always stick with `image` (though the command line input could stay as `-t` I guess only because `-i` is taken. Also I started to make this change on my branch, but I renamed these options to just `binary`, `embed`, etc. Seems more directive... So this could be `embedImages`.